src/types.jl

```diff
@@ -163,11 +163,11 @@ function AnnotationsSignal(records::Vector{Vector{TimestampedAnnotationList}})
     # treated similarly, i.e. the `SignalHeader` is overly trusted).
     max_bytes_per_record = maximum(sum(write_tal(IOBuffer(), tal) for tal in record)
                                    for record in records)
-    return AnnotationsSignal(Int16(cld(max_bytes_per_record, 2)), records)
+    return AnnotationsSignal(Int32(cld(max_bytes_per_record, 2)), records)
 end
 
 function SignalHeader(signal::AnnotationsSignal)
-    return SignalHeader("EDF Annotations", "", "", -1, 1, -32768, 32767,
+    return SignalHeader("EDF Annotations", "", "", -1, 1, typemin(Int16), typemax(Int16),
```

Comment:
this is actually from the spec:
> For the sake of EDF compatibility, the fields 'digital minimum' and 'digital maximum' must be filled with -32768 and 32767, respectively.

Section 2.2.1, Para 2, Sentence 2 https://www.edfplus.info/specs/edfplus.html#additionalspecs

```suggestion
    return SignalHeader("EDF Annotations", "", "", -1, 1, -32768, 32767,
```

Reply:
These are the same exact numbers (`typemin(Int16) == -32768`). Seems better to communicate the intent of the numbers over the literal values.

Reply:
I think the literals are better since they are in the spec, so it makes it easy to check for compliance against the spec

Reply:
> These are the same exact numbers (typemin(Int16) == -32768). Seems better to communicate the intent of the numbers over the literal values.

The intent of hte values is to follow the spec, which specifies them literally.

Reply:
in any case this change is unrelated to expanding the "samples per record".